src/main/java/com/thevoxelbox/voxelsniper/brush/type/canyon/CanyonBrush.java

```diff
@@ -7,8 +7,12 @@
 import com.thevoxelbox.voxelsniper.brush.type.AbstractBrush;
 import com.thevoxelbox.voxelsniper.sniper.snipe.Snipe;
 import com.thevoxelbox.voxelsniper.sniper.snipe.message.SnipeMessenger;
+import com.thevoxelbox.voxelsniper.util.text.NumericParser;
 import org.bukkit.ChatColor;
 
+import java.util.List;
+import java.util.stream.Stream;
+
 public class CanyonBrush extends AbstractBrush {
```

Comment:
I'm unsure whether this should be addressed here or in Fawe, however, it doesn't calculate light properly
![](https://i.imgur.com/1aCyqQU.jpg)

Reply:
_Huphm, no idea again, light issues_ 😭
Works fine on Tuinity (except on chunk edges, whereas it should), seems FAWE implementation related 🤷 
![2021-08-12_23 54 48](https://user-images.githubusercontent.com/43724816/129275118-4d57c5ee-3d00-4965-a042-6a1902eaed8b.png)

Reply:
Meh, I'll ignore this for now. We have https://github.com/IntellectualSites/FastAsyncWorldEdit-Adapters/issues/6 in place.